config.js

```diff
@@ -1490,4 +1490,7 @@ var config = {
     //      'modules/statistics/CallStats.js': 'info',
     //      'modules/xmpp/strophe.util.js': 'log',
     // },
+
+    // Application logo url
+    defaultLogoUrl: 'images/watermark.svg',
```

Comment:
We need this commented and the default in the code. 

The debian installations on update do not update config.js and this will break their deployments.

Reply:
updated